packages/related-content/style.module.css

```diff
@@ -0,0 +1,63 @@
+.wrapper {
+  composes: g-grid from global;
+  composes: g-grid-container from global;
+  &.dark {
+    background-color: var(--wpl-neutral-900);
+  }
+}
```

Comment:
Is it up to the consumer of the component to include the vertical padding of this component?

If so, I would leave the background color up to their decision as well. Since they'd have to apply the same background color you are applying here as well in dark appearance.

Reply:
Added vertical padding. 👍